audformat/core/table.py

```diff
@@ -770,6 +813,51 @@ class MiscTable(Base):
         file other
         f1   f2     True
              f3    False
+        >>> index_ex = pd.MultiIndex.from_tuples(
+        ...   [
+        ...     ('f4', 'f1'),
+        ...   ],
+        ...   names=['file', 'other'],
+        ... )
+        >>> index_ex = utils.set_index_dtypes(index_ex, 'string')
+        >>> table_ex = table.extend_index(
+        ...     index_ex,
+        ...     inplace=False,
+        ... )
+        >>> table_ex.get()
+                    match
+        file other
+        f1   f2      True
+             f3     False
+        f2   f3      True
+        f4   f1       NaN
+        >>> table_ex.set(
+        ...     {'match': True},
+        ...     index=index_ex,
```

Comment:
Here I was wondering first as I expected that `index_ex` would cover all entries in `table_ex` as it has the same name, but this is not the case.
But we have the same example for `Table`, so maybe we just stick with it.

Reply:
Yes, it's slightly confusing, maybe rename `index_ex` to `index_2`?

Reply:
Yes, please rename to `index_new` in the example for `MiscTable` and `Table`

Reply:
done